tests/ui/suggestions/issue-84973-blacklist.stderr

```diff
@@ -66,6 +66,11 @@ note: required by a bound in `f_sized`
    |
 LL | fn f_sized<T: Sized>(t: T) {}
    |               ^^^^^ required by this bound in `f_sized`
+help: references to `!Sized` types like `&dyn Fn()` are `Sized`; consider not dereferencing the expression
```

Comment:
I find this wording to be a bit garden-path-y. Perhaps:

> References are always `Sized`, even if they point to unsized data; consider not dereferencing the expression

Reply:
The reason I think it's garden path-y is because it looks like you're saying `&dyn Fn()` is `!Sized`.